src/routes/metadata.ts

```diff
@@ -7,12 +7,14 @@ import { setMostRecentPing } from 'services/heartBeat';
 import { getNumFramesFromChunkName } from 'util/framekm';
 import { join } from 'path';
 import { promisify } from 'util';
+import { ensureAclPassed } from './middleware/acl';
 
 const MAX_RESPONSE_SIZE = 10000;
 const router = Router();
 
-router.get('/', async (req: Request, res: Response) => {
+router.get('/', ensureAclPassed, async (req: Request, res: Response) => {
   try {
+    console.log('[METADATA] Control reached to metadata route fetching the data');
```

Comment:
If you don't mind, 
let's remove this one just in case,
App calls this pretty frequently, it will make the dashcam logs very messy